bin/thoth

```diff
@@ -50,26 +50,25 @@ var thoth = {
 	logs: 'logs',
 	log: 'log',
 	verbose: true,
-	asset: 'thoth.tar.gz',
+	asset: '/thoth/public/thoth-sunos-' + thothVersion + '.tar.gz',
 	db: {
 		host: 'localhost',
 		port: 28015,
 		db: '',
 		authKey: 'bagnoogle',
 		table: 'dumps'
 	},
-	indexed: {
+	sysprops: {
 		name: true,
-		time: true,
-		type: true,
-		cmd: true,
-		node: true,
 		dump: true,
-		ticket: true
+		platform: true,
+		node: true,
+		version: true,
+		type: true,
+		time: true,
+		stack: true
 	},
-	useindex: true,
-	analyzerdebug: false,
-	v8: '/Joyent_Dev/public/mdb_v8/latest',
+	jobby: undefined,
```

Comment:
If you *want*, I have a patch to add support for a `THOTH_NO_JOBS=1` envvar telling `thoth` to use the non-jobby code path so that you can do this against the thoth in current JPC Manta for example:

```
[root@5d4f7599-a991-6b35-dd44-d91936957a6b ~/joy/manta-thoth]# THOTH_NO_JOBS=1 thoth debug f824873602b1c53d
thoth: downloading core.node.97708 to local cache
...
```

Patch: https://gist.github.com/trentm/34837fbccf0fb108d06b5c4d59f6407c
No problem at all if you don't want to support this.

Reply:
thanks!